common/deposits/hfe_deposit_features.txt

```diff
@@ -15,6 +15,91 @@ hfe_deposit_features_spiral_of_death_0 = {
 			consumer_goods = 5
 		}
 	}
+	drop_weight = {
+		weight = 0
+	}
+}
+
+HFE_TSP_exodus_start_point = {
```

Comment:
I guess this is for habitable planet.
Then its a problem.

Reply:
No, it's a Deposit for a _Shattered World_.